test/services/fileStorage/aws/awsS3.test.js

```diff
@@ -2,6 +2,7 @@ const chai = require('chai');
 const chaiHttp = require('chai-http');
 const { expect } = require('chai');
 const mockery = require('mockery');
+const { Configuration } = require('@schul-cloud/commons');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Configuration' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5222818)